### ISSUE TITLE
Add leaflet-fullHash.js . 

### ISSUE THREAD

**Issue by sagarpreet-chadha** [publiclab/mapknitter#510]
Continued from : https://github.com/publiclab/mapknitter/issues/419#issuecomment-481160685

Let's add` leaflet-fullHash.js` to mapknitter index map 🗺 .

- [x] Add leaflet-fullHash.js to bower.json .  See this for example :
https://github.com/publiclab/leaflet-environmental-layers/blob/cb4b72824d0186bba1fa1bca1bdafff8a1742992/package.json#L46

- [x] Incorporate this dependency in the from of script tag in `view/maps/index.html.erb` .
See this for example : https://github.com/publiclab/leaflet-environmental-layers/blob/cb4b72824d0186bba1fa1bca1bdafff8a1742992/example/index.html#L47

- [ ] Add leaflet-fullHash object code in `view/maps/index.html.erb` , see this for example : 
https://github.com/publiclab/leaflet-environmental-layers/blob/cb4b72824d0186bba1fa1bca1bdafff8a1742992/example/index.html#L333

We currently use this library in Leaflet environmental Layers and plots2 😄 . Lets use this in mapknitter as well 🎈 !

**Comment by divyabaid16** [comment]
Can I help here?

**Comment by igniteeng000** [comment]
Hi @sagarpreet-chadha . I have already solved this issue, Please check https://github.com/publiclab/mapknitter/pull/501.

**Comment by igniteeng000** [comment]
But, I have added leaflet hash instead of leaflet-full Hash.  I will send another PR soon.

**Comment by divyabaid16** [comment]
Oops!!
I was about to send the PR.
No problem!! I will look into other issue :smile:

**Comment by gauravano** [comment]
Hey @sagarpreet-chadha, the PR for the first part is up at #516, please review. Thanks!

**Comment by sashadev-sky** [comment]
@sagarpreet-chadha reopening this as point 3 has not been resolved yet. Is it still relevant and, if so,
would you like to open an FTO with the code to add the new leaflet-fullHash object?

**Comment by sagarpreet-chadha** [comment]
Opened issue #621 for solving 3rd pointer 👍

**Comment by sashadev-sky** [comment]
@sagarpreet-chadha great thank you!! i assigned it to someone already 😊

**Comment by gauravano** [comment]
Solved in https://github.com/publiclab/mapknitter/pull/891. @kaustubh-nair @alaxalves please verify. Thank you!

**Comment by sashadev-sky** [comment]
@gauravano still not positive what the problem / ideal outcome was here so you might be a better a better judge on this --- what about #621 ?? If there's anything left to merge we should do that ! And I guess if we have to close it, close it

**Comment by sashadev-sky** [comment]
@sagarpreet-chadha

**Comment by sagarpreet-chadha** [comment]
Hi @sashadev-sky ,
sorry for reverting back late. 
I found something weird, mapknitter.org does not have the LEL layers! 
It is in the codebase:  https://github.com/publiclab/mapknitter/blob/ab4727f085a507f469cb27ebd97f5a44b5cfe9a8/app/views/maps/index.html.erb#L236

But not showing in the production .

And these lines are not working as you can check on website: 
https://github.com/publiclab/mapknitter/blob/ab4727f085a507f469cb27ebd97f5a44b5cfe9a8/app/views/maps/index.html.erb#L123

@jywarren , Is production pointing to different branch?

**Comment by jywarren** [comment]
Was it perhaps installed on some maps but not others? We should have it
installed on the new View Only page that Cess is working on, for sure. And
perhaps on the front page tour map. Could we integrate it with the new
one-line all layers function once that is done?

**Comment by sagarpreet-chadha** [comment]
Okay, yeah makes sense . Lets preempt integration till the new one-liner feature is ready . Thanks
